pod.json

```diff
@@ -12,7 +12,7 @@
           "ports": [{
               "containerPort": 8443,
           }],
-          "command": ["start", "master", "--kubernetes=:8443"],
```

Comment:
Also, this changes from a master to an all-in-one... not sure we want an all in one running in a pod